src/main/drivers/accgyro/accgyro.h

```diff
@@ -77,7 +77,8 @@ typedef struct gyroDev_s {
     uint8_t hardware_32khz_lpf;
     uint8_t mpuDividerDrops;
     ioTag_t mpuIntExtiTag;
-    uint8_t filler[2];
+    bool gyroHasOverflowProtection;
```

Comment:
`sizeof(boolean)` is not guaranteed to be 1, use `uint8_t` instead.

Reply:
In the structure there are other `bool` entries and even a conditional compilation for `SIMULATOR_BUILD` so keeping alignment seems difficult.  The padding as 2 before my addition and I changed to 1 to keep the alignment.  Removing the padding results in `warning: padding struct size to alignment boundary [-Wpadded]` warnings.

Reply:
I still suggest using `uint8_t` for new members

Reply:
Updated per suggestion.